docs/guidance/example.md

```diff
@@ -62,28 +62,30 @@ Each project object contains the following sections:
 
 This section provides contextual information about the infrastructure project, including:
 
+* a project identifier in the `id` field. To make the project identifier globally unique, a project identifier prefix must be added to the local identifier for the project. Project identifier prefixes are assigned by the OC4IDS helpdesk. For more information on project identifiers, refer to the [project identifiers guidance](https://github.com/open-contracting/infrastructure/pull/159/identifiers.md).
```

Comment:
Please correct the URL to the project identifiers guidance.

I think Github did something clever with my proposed text and changed the relative URL to an absolute one for the PR (I should have wrapped it in backticks)